cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -339,20 +365,37 @@ void CobFrameTracker::publishHoldTwist(const ros::Duration& period)
     }
     else
     {
-        ROS_ERROR_STREAM_THROTTLE(1, "Abortion active: Publishing hold posture twist");
         if (success)
         {
-            twist_msg.twist.linear.x = pid_controller_trans_x_.computeCommand(ht_.transform_tf.getOrigin().x() - transform_tf.getOrigin().x(), period);
-            twist_msg.twist.linear.y = pid_controller_trans_y_.computeCommand(ht_.transform_tf.getOrigin().y() - transform_tf.getOrigin().y(), period);
-            twist_msg.twist.linear.z = pid_controller_trans_z_.computeCommand(ht_.transform_tf.getOrigin().z() - transform_tf.getOrigin().z(), period);
-
-            twist_msg.twist.angular.x = pid_controller_rot_x_.computeCommand(ht_.transform_tf.getRotation().x() - transform_tf.getRotation().x(), period);
-            twist_msg.twist.angular.y = pid_controller_rot_y_.computeCommand(ht_.transform_tf.getRotation().y() - transform_tf.getRotation().y(), period);
-            twist_msg.twist.angular.z = pid_controller_rot_z_.computeCommand(ht_.transform_tf.getRotation().z() - transform_tf.getRotation().z(), period);
+            ROS_ERROR_STREAM_THROTTLE(1, "Abortion active: Publishing hold posture twist");
+
+            double error_trans_x = ht_.transform_tf.getOrigin().x() - transform_tf.getOrigin().x();
+            double error_trans_y = ht_.transform_tf.getOrigin().y() - transform_tf.getOrigin().y();
+            double error_trans_z = ht_.transform_tf.getOrigin().z() - transform_tf.getOrigin().z();
+            double error_rot_x = ht_.transform_tf.getRotation().x() - transform_tf.getRotation().x();
+            double error_rot_y = ht_.transform_tf.getRotation().y() - transform_tf.getRotation().y();
+            double error_rot_z = ht_.transform_tf.getRotation().z() - transform_tf.getRotation().z();
+
+            error_msg.twist.linear.x = error_trans_x;
+            error_msg.twist.linear.y = error_trans_y;
+            error_msg.twist.linear.z = error_trans_z;
+            error_msg.twist.angular.x = error_rot_x;
+            error_msg.twist.angular.y = error_rot_y;
+            error_msg.twist.angular.z = error_rot_z;
+
+            twist_msg.twist.linear.x = pid_controller_trans_x_.computeCommand(error_trans_x, period);
+            twist_msg.twist.linear.y = pid_controller_trans_y_.computeCommand(error_trans_y, period);
+            twist_msg.twist.linear.z = pid_controller_trans_z_.computeCommand(error_trans_z, period);
+
+            twist_msg.twist.angular.x = pid_controller_rot_x_.computeCommand(error_rot_x, period);
+            twist_msg.twist.angular.y = pid_controller_rot_y_.computeCommand(error_rot_y, period);
+            twist_msg.twist.angular.z = pid_controller_rot_z_.computeCommand(error_rot_z, period);
         }
     }
 
+    ROS_INFO_STREAM("[publishHoldTwist] Period: " << period.toSec());
```

Comment:
@ipa-aep there is a second occurrence of `ROS_INFO_STREAM`...please remove this, too!